tests/test_end_to_end.py

```diff
@@ -1,15 +1,22 @@
 import importlib
+import os
 import sys
+from unittest import mock
 
 import pytest
 
 import hamilton.driver
 import tests.resources.data_quality
-
-# import tests.resources.smoke_screen_module
+import tests.resources.dynamic_config
 from hamilton.data_quality.base import DataValidationError, ValidationResult
 
 
+@pytest.fixture(autouse=True)
+def power_user_mode():
+    with mock.patch.dict(os.environ, {"HAMILTON_POWER_USER_MODE": "true"}):
+        yield
```

Comment:
should we make it a constructor argument to the driver?

That way when someone is reading the code, they see that driver is taking in the power mode variable? 

My concern here is that someone could turn this on or off and a user wouldn't necessarily know what's changed.

Reply:
On second thought, the error message is pretty clear, and doing it on the driver-level is weird --  it is an inherent property of the module, rather than the driver. We can always add it to the driver to turn it on, but I think it's pretty easy to see what's happening, and will end up in the driver script anyway.